### ISSUE TITLE
Rulesets cannot be loaded in 52.0.3

### ISSUE THREAD

**Issue by jtarvainen** [gajus/eslint-plugin-jsdoc#1443]
## Expected behavior

As described in the [documentation](https://www.npmjs.com/package/eslint-plugin-jsdoc), rulesets can be loaded from a `.js` config file as follows:

```
module.exports = {
    extends: [
        'plugin:jsdoc/recommended'
    ]
};
```

## Actual behavior

After updating `eslint-plugin-jsdoc` to v52.0.3, rulesets can no longer be loaded:

> ESLint couldn't find the config "plugin:jsdoc/recommended" to extend from. Please check that the name of the config is correct.

## ESLint Config

See example above.

## Environment

- Node version: v22.18.0
- ESLint version v8.57.1
- `eslint-plugin-jsdoc` version: v52.0.3

**Comment by brettz9** [comment]
I'm working on a reversion now. It seems like the `BREAKING CHANGES:` formatting didn't trigger a major bump either.

**Comment by brettz9** [comment]
> Can somebody downgrade back to 52.0.3 and see if this commit fixes it? [4e69bdb](https://github.com/gajus/eslint-plugin-jsdoc/commit/4e69bdb88472c1047fa2730d2070c653b9eee10c) My hunch is that ESLint 8 might not support `exports`.

Sorry, no, the issue was that it needed default exports in the main `src/index.js` file. But changing to a default export for that file (and adding separate config for the other two files) led to other tsdown validation problems, and I didn't have the bandwidth to try to resolve.

**Comment by lishaduck** [comment]
> the issue was that it needed default exports in the main `src/index.js` file. But changing to a default export for that file (and adding separate config for the other two files) led to other tsdown validation problems, and I didn't have the bandwidth to try to resolve.

I'm lost. It does use a default export?
If you still have the branch, could you push it up somewhere I could take a look?

**Comment by brettz9** [comment]
> > the issue was that it needed default exports in the main `src/index.js` file. But changing to a default export for that file (and adding separate config for the other two files) led to other tsdown validation problems, and I didn't have the bandwidth to try to resolve.
> 
> I'm lost. It does use a default export? 

It should produce a default export for CommonJS (i.e., to produce `module.exports`)

> If you still have the branch, could you push it up somewhere I could take a look?

https://github.com/gajus/eslint-plugin-jsdoc/compare/main...brettz9:eslint-plugin-jsdoc:tsdown
